modules/social_features/social_event/src/Plugin/GraphQL/QueryHelper/EventManagersQueryHelper.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Drupal\social_event\Plugin\GraphQL\QueryHelper;
+
+use Drupal\Core\Database\Connection;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Entity\Query\QueryInterface;
+use Drupal\graphql\GraphQL\Buffers\EntityBuffer;
+use Drupal\node\NodeInterface;
+use Drupal\social_graphql\GraphQL\ConnectionQueryHelperBase;
+use Drupal\social_graphql\Wrappers\Cursor;
+use Drupal\social_graphql\Wrappers\Edge;
+use Drupal\user\UserInterface;
+use GraphQL\Deferred;
+use GraphQL\Executor\Promise\Adapter\SyncPromise;
+
+/**
+ * Loads event managers.
+ */
+class EventManagersQueryHelper extends ConnectionQueryHelperBase {
+
+  /**
+   * The event for which managers are being fetched.
+   *
+   * @var \Drupal\node\NodeInterface
+   */
+  protected NodeInterface $event;
+
+  /**
+   * The database service.
+   *
+   * @var \Drupal\Core\Database\Connection
+   */
+  protected Connection $database;
+
+  /**
+   * EventManagersQueryHelper constructor.
+   *
+   * @param string $sort_key
+   *   The key that is used for sorting.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The Drupal entity type manager.
+   * @param \Drupal\graphql\GraphQL\Buffers\EntityBuffer $graphql_entity_buffer
+   *   The GraphQL entity buffer.
+   * @param \Drupal\node\NodeInterface $event
+   *   The event.
+   * @param \Drupal\Core\Database\Connection $database
+   *   The database.
+   */
+  public function __construct(
+    string $sort_key,
+    EntityTypeManagerInterface $entity_type_manager,
+    EntityBuffer $graphql_entity_buffer,
+    NodeInterface $event,
+    Connection $database
+  ) {
+    parent::__construct($sort_key, $entity_type_manager, $graphql_entity_buffer);
+
+    $this->event = $event;
+    $this->database = $database;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getQuery() : QueryInterface {
+    $query = $this->database->select('node__field_event_managers', 'fem');
+    $query->addField('fem', 'field_event_managers_target_id');
+    $query->condition('entity_id', $this->event->id());
+    $query->condition('revision_id', $this->event->getLoadedRevisionId());
+    $uids = $query->execute()->fetchCol();
+
+    return $this->entityTypeManager->getStorage('user')
+      ->getQuery()
+      ->accessCheck(TRUE)
+      ->condition('uid', $uids ?: NULL, 'IN');
```

Comment:
Can we be more efficient here? Similarly to what we do for participants in chats (see `ConversationParticipantsQueryHelper`).

Perhaps since we already have a loaded event here we can just get the field value, rather than making a custom database query? 

Will this break if there are no event managers? I think `IN NULL` is not valid right?

Reply:
So, we have an `event` node in QueryHelper, but we should return the `user` query, another way that I see, can be something like:
```
    $users = $this->event->field_event_managers->referencedEntities();
    $uids = array_map(function ($entity) {
      return $entity->id();
    }, $users);

    return $this->entityTypeManager->getStorage('user')
      ->getQuery()
      ->accessCheck(TRUE)
      ->condition('uid', $uids ?: NULL, 'IN');
```

P.S. It will not break the query :)

Reply:
For now, added my new proposal.